src/libraries/MarketParamsLib.sol

```diff
@@ -12,7 +12,7 @@ library MarketParamsLib {
     /// @dev The length is 5 * 32 because `MarketParams` has 5 variables of 32 bytes each.
     uint256 internal constant MARKET_PARAMS_BYTES_LENGTH = 5 * 32;
 
-    /// @notice Returns the id of the market `marketParams`.
+    /// @notice Returns the id (`marketParamsId`) of the market `marketParams`.
```

Comment:
I actually think it's clearer without this ^^

Reply:
will remove then